sdk2ecr

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+set -e -o pipefail
+shopt -qs failglob
+
+for opt in "$@"; do
+   optarg="$(expr "${opt}" : '[^=]*=\(.*\)')"
+   case "${opt}" in
+      --version=*) VERSION="${optarg}" ;;
+      --registry=*) REGISTRY="${optarg}" ;;
+      --sdk-name=*) SDK_NAME="${optarg}" ;;
+   esac
+done
+
+if [ -z "${VERSION}" ]; then
+  VERSION=$(make print-version | tail -1)
+fi
```

Comment:
I'd prefer to just treat `VERSION` as required too - it's odd to have a tool called by `make` that calls back into `make`.